### PR TITLE
[IMP]Export Wizard: Improve code to not remove fields from 'Fields to…

### PR DESCRIPTION
**Pull request by rba-odoo**
… Export'on Export all data and remove noncompatible field on selecting Import Compatible Option

Task: https://www.odoo.com/web#id=5110&view_type=form&model=project.task&action=327
Pad: https://pad.odoo.com/p/r.lZVj53TW0aZDnp2v
Issue: https://www.odoo.com/web#id=644493&view_type=form&model=project.issue&menu_id=4120&action=609

### CODE REVIEW

File: addons/web/static/src/js/widgets/data_export.js [hunk 1]
```diff
@@ -55,13 +56,23 @@ var DataExport = Dialog.extend({
...
-            self.$el.find('#fields_list').empty();
             self.$el.find('#field-tree-structure').remove();
             var import_comp = self.$el.find("#import_compat").val();
             self.rpc("/web/export/get_fields", {
                 model: self.dataset.model,
                 import_compat: !!import_comp,
             }).done(function (records) {
+                if (import_comp) {
+                    var export_list = self.$("#fields_list option").map(function (res) {
+                        console.log(this);
```
Comment (xmo-odoo) [inline addons/web/static/src/js/widgets/data_export.js]: leftover console call

File: addons/web/static/src/js/widgets/data_export.js [hunk 2]
```diff
@@ -55,13 +56,23 @@ var DataExport = Dialog.extend({
...
+                        for (var i = 0; i < records.length; i++) {
+                            if (this.value === records[i].id || $(this).attr("import_compatible") === "false") {break;}
+                        }
```
Comment (xmo-odoo) [inline addons/web/static/src/js/widgets/data_export.js]: why not just use `_.find`? or `_.findWhere` to see if there's a record whose id matches `this.value`?

File: addons/web/static/src/js/widgets/data_export.js (hunk 1 above)
Comment (rba-odoo) [inline addons/web/static/src/js/widgets/data_export.js]: oppss.. forgot it. Removed .!

File: addons/web/static/src/js/widgets/data_export.js (hunk 2 above)
Comment (rba-odoo) [inline addons/web/static/src/js/widgets/data_export.js]: @xmo-odoo  okay, Done.!

**Comment by rim-odoo** [comment]
@xmo-odoo Can I have your opinion on this fix?

**Comment by xmo-odoo** [comment]
@rim-odoo I don't have one.

**Pull request closed without merging**